qcodes/tests/drivers/test_tektronix_AWG70000A.py

```diff
@@ -185,7 +185,7 @@ def test_seqxfile_from_fs(forged_sequence):
     path_to_schema = auxfiles.__file__.replace('__init__.py',
                                                'awgSeqDataSets.xsd')
 
-    with open(path_to_schema, 'r') as fid:
+    with open(path_to_schema) as fid:
```

Comment:
While this is identical I am not convinsed that removing arguments just because they are the default value is an improvment

Reply:
agree with Jens.

Reply:
> I am not convinced that removing arguments just because they are the default value is an improvement

It doesn't always do this, just with `open` AFAIK (where I think it's fine).

Unfortunately, this option isn't configurable.